smac/epm/gaussian_process/gpytorch.py

```diff
@@ -104,6 +108,12 @@ def __init__(
             The number of components to keep when using PCA to reduce dimensionality of instance features. Requires to
             set n_feats (> pca_dims).
         """
+        if _scipy_objective_and_grad is None:
```

Comment:
Since we have a this function under our own codebase, I will simply delete this check